src/pages/IdeLoader/index.tsx

```diff
@@ -295,6 +296,7 @@ class IdeLoader extends React.PureComponent<Props, State> {
 
     return (
       <React.Fragment>
+        <Head pageName="Workspace Loader" />
```

Comment:
I don't have a strong opinion here but maybe something like `Loading Workspace` or even `Loading WSNAME` would be better, currently, we have just `workspace name` which is good as well.

Reply:
I have fixed it